test/init_test.go

```diff
@@ -22,165 +22,23 @@ limitations under the License.
 package test
 
 import (
-	"context"
 	"flag"
 	"fmt"
 	"os"
-	"strings"
-	"sync"
 	"testing"
 
-	"github.com/tektoncd/pipeline/pkg/apis/config"
-	"github.com/tektoncd/pipeline/pkg/names"
-	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/util/wait"
-	"k8s.io/client-go/kubernetes"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp" // Mysteriously by k8s libs, or they fail to create `KubeClient`s when using oidc authentication. Apparently just importing it is enough. @_@ side effects @_@. https://github.com/kubernetes/client-go/issues/345
 	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
-	"knative.dev/pkg/system"
 	knativetest "knative.dev/pkg/test"
-	"knative.dev/pkg/test/logging" // Mysteriously by k8s libs, or they fail to create `KubeClient`s from config. Apparently just importing it is enough. @_@ side effects @_@. https://github.com/kubernetes/client-go/issues/242
-	"knative.dev/pkg/test/logstream"
-	"sigs.k8s.io/yaml"
+	// Mysteriously by k8s libs, or they fail to create `KubeClient`s from config. Apparently just importing it is enough. @_@ side effects @_@. https://github.com/kubernetes/client-go/issues/242
```

Comment:
This is because they use `init()` function, a piece of code that is executed when you import 😅. This is.. usually a bad pattern to use in libraries 🙃 

But I realize this comment "might" be misplaced or not needed any more.

Reply:
Thanks for the catch;)